Tests fail with recent protobuf versions

libprotoc 3.21.12
python protobuf 4.21.12

```
test setup failed
@pytest.fixture
    def empty_history_response() -> HistoryResponse:
>       return mock_history_response(time_delta=[], aggregate=[], value=[])

../../../../metricq/metricq-python/tests/test_history_client.py:32: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../../metricq/metricq-python/tests/test_history_client.py:24: in mock_history_response
    proto = create_autospec(
/usr/lib/python3.10/unittest/mock.py:2717: in create_autospec
    mock = Klass(parent=_parent, _new_parent=_parent, _new_name=_new_name,
/usr/lib/python3.10/unittest/mock.py:2111: in __init__
    _safe_super(MagicMixin, self).__init__(*args, **kw)
/usr/lib/python3.10/unittest/mock.py:1096: in __init__
    _safe_super(CallableMixin, self).__init__(
/usr/lib/python3.10/unittest/mock.py:466: in __init__
    self.configure_mock(**kwargs)
/usr/lib/python3.10/unittest/mock.py:635: in configure_mock
    setattr(obj, final, val)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <MagicMock spec_set='HistoryResponse' id='140006628651360'>
name = 'time_delta', value = []

    def __setattr__(self, name, value):
        if name in _allowed_names:
            # property setters go through here
            return object.__setattr__(self, name, value)
        elif (self._spec_set and self._mock_methods is not None and
            name not in self._mock_methods and
            name not in self.__dict__):
>           raise AttributeError("Mock object has no attribute '%s'" % name)
E           AttributeError: Mock object has no attribute 'time_delta'

/usr/lib/python3.10/unittest/mock.py:759: AttributeError

```

Apparently some magic in the protobuf objects was changed so it cannot be used as autospec.

--- a/tests/test_history_client.py
+++ b/tests/test_history_client.py
@@ -1,5 +1,4 @@
 from typing import Any
-from unittest.mock import create_autospec
 
 import pytest
 from pytest_mock import MockerFixture
@@ -21,10 +20,9 @@ DEFAULT_METRIC = "test.foo"
 
 def mock_history_response(**response_fields: Any) -> HistoryResponse:
     response_fields.setdefault("error", "")
-    proto = create_autospec(
-        history_pb2.HistoryResponse, spec_set=True, **response_fields
+    return HistoryResponse(
+        proto=history_pb2.HistoryResponse(**response_fields),
     )
-    return HistoryResponse(proto=proto)
 
 
 @pytest.fixture
@@ -55,19 +53,19 @@ def test_iterate_empty_history_response(
 async def test_history_aggregate(
     history_client: HistoryClient, mocker: MockerFixture
 ) -> None:
-    TIME = Timestamp(0)
-    AGGREGATE = create_autospec(history_pb2.HistoryResponse.Aggregate, spec_set=True)
+    time = Timestamp(0)
+    aggregate = history_pb2.HistoryResponse.Aggregate()
 
     response = mock_history_response(
-        time_delta=[TIME.posix_ns],
-        aggregate=[AGGREGATE],
+        time_delta=[time.posix_ns],
+        aggregate=[aggregate],
     )
 
     patch_history_data_request(mocker, response)
 
     assert await history_client.history_aggregate(
         DEFAULT_METRIC
-    ) == TimeAggregate.from_proto(timestamp=TIME, proto=AGGREGATE)
+    ) == TimeAggregate.from_proto(timestamp=time, proto=aggregate)
 
 
 async def test_history_no_aggregate(
@@ -95,18 +93,18 @@ async def test_history_error(
 async def test_history_last_value(
     history_client: HistoryClient, mocker: MockerFixture
 ) -> None:
-    TIME = Timestamp(0)
-    VALUE = mocker.sentinel.VALUE
+    time = Timestamp(0)
+    value = 42.23
 
     response = mock_history_response(
-        time_delta=[TIME.posix_ns],
-        value=[VALUE],
+        time_delta=[time.posix_ns],
+        value=[value],
     )
 
     patch_history_data_request(mocker, response)
 
     assert await history_client.history_last_value(DEFAULT_METRIC) == TimeValue(
-        timestamp=TIME, value=VALUE
+        timestamp=time, value=value
     )
 
 
@@ -138,17 +136,17 @@ async def test_get_metrics_historic_only(
     like :meth:`metricq.Client.get_metrics` by default, except that
     :code:`historic=True` is passed.
     """
-    RESULT = mocker.sentinel.RESULT
+    result = mocker.sentinel.RESULT
 
     async def assert_historic_true(
         self: HistoryClient, *args: Any, historic: bool, **kwargs: Any
     ) -> Any:
         assert historic
-        return RESULT
+        return result
 
     mocker.patch("metricq.Client.get_metrics", assert_historic_true)
 
-    assert await history_client.get_metrics(selector=DEFAULT_METRIC) is RESULT
+    assert await history_client.get_metrics(selector=DEFAULT_METRIC) is result
 
 
 @pytest.mark.parametrize(("historic_override"), [True, False])

